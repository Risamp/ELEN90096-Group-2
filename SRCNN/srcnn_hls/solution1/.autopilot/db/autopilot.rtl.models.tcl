set SynModuleInfo {
  {SRCNAME srcnn_Pipeline_1 MODELNAME srcnn_Pipeline_1 RTLNAME srcnn_srcnn_Pipeline_1
    SUBMODULES {
      {MODELNAME srcnn_flow_control_loop_pipe_sequential_init RTLNAME srcnn_flow_control_loop_pipe_sequential_init BINDTYPE interface TYPE internal_upc_flow_control INSTNAME srcnn_flow_control_loop_pipe_sequential_init_U}
    }
  }
  {SRCNAME srcnn_Pipeline_2 MODELNAME srcnn_Pipeline_2 RTLNAME srcnn_srcnn_Pipeline_2}
  {SRCNAME srcnn_Pipeline_3 MODELNAME srcnn_Pipeline_3 RTLNAME srcnn_srcnn_Pipeline_3}
  {SRCNAME conv1 MODELNAME conv1 RTLNAME srcnn_conv1
    SUBMODULES {
      {MODELNAME srcnn_mul_5ns_6ns_10_1_1 RTLNAME srcnn_mul_5ns_6ns_10_1_1 BINDTYPE op TYPE mul IMPL auto LATENCY 0 ALLOW_PRAGMA 1}
      {MODELNAME srcnn_conv1_input_fm_buffer_2_0_RAM_AUTO_1R1W RTLNAME srcnn_conv1_input_fm_buffer_2_0_RAM_AUTO_1R1W BINDTYPE storage TYPE ram IMPL auto LATENCY 2 ALLOW_PRAGMA 1}
      {MODELNAME srcnn_conv1_output_fm_buffer_1_RAM_AUTO_1R1W RTLNAME srcnn_conv1_output_fm_buffer_1_RAM_AUTO_1R1W BINDTYPE storage TYPE ram IMPL auto LATENCY 2 ALLOW_PRAGMA 1}
    }
  }
  {SRCNAME load_buffer_tile_c2_Pipeline_1 MODELNAME load_buffer_tile_c2_Pipeline_1 RTLNAME srcnn_load_buffer_tile_c2_Pipeline_1}
  {SRCNAME load_buffer_tile_c2_Pipeline_2 MODELNAME load_buffer_tile_c2_Pipeline_2 RTLNAME srcnn_load_buffer_tile_c2_Pipeline_2}
  {SRCNAME load_buffer_tile_c2_Pipeline_VITIS_LOOP_105_1 MODELNAME load_buffer_tile_c2_Pipeline_VITIS_LOOP_105_1 RTLNAME srcnn_load_buffer_tile_c2_Pipeline_VITIS_LOOP_105_1
    SUBMODULES {
      {MODELNAME srcnn_mul_6ns_19ns_24_1_1 RTLNAME srcnn_mul_6ns_19ns_24_1_1 BINDTYPE op TYPE mul IMPL auto LATENCY 0 ALLOW_PRAGMA 1}
    }
  }
  {SRCNAME load_buffer_tile_c2_Pipeline_VITIS_LOOP_121_4 MODELNAME load_buffer_tile_c2_Pipeline_VITIS_LOOP_121_4 RTLNAME srcnn_load_buffer_tile_c2_Pipeline_VITIS_LOOP_121_4}
  {SRCNAME load_buffer_tile_c2 MODELNAME load_buffer_tile_c2 RTLNAME srcnn_load_buffer_tile_c2}
  {SRCNAME conv2_Pipeline_1 MODELNAME conv2_Pipeline_1 RTLNAME srcnn_conv2_Pipeline_1}
  {SRCNAME conv2 MODELNAME conv2 RTLNAME srcnn_conv2
    SUBMODULES {
      {MODELNAME srcnn_conv2_input_fm_buffer_1_RAM_AUTO_1R1W RTLNAME srcnn_conv2_input_fm_buffer_1_RAM_AUTO_1R1W BINDTYPE storage TYPE ram IMPL auto LATENCY 2 ALLOW_PRAGMA 1}
      {MODELNAME srcnn_conv2_weights_buffer_0_0_0_RAM_AUTO_1R1W RTLNAME srcnn_conv2_weights_buffer_0_0_0_RAM_AUTO_1R1W BINDTYPE storage TYPE ram IMPL auto LATENCY 2 ALLOW_PRAGMA 1}
      {MODELNAME srcnn_conv2_output_fm_buffer_RAM_AUTO_1R1W RTLNAME srcnn_conv2_output_fm_buffer_RAM_AUTO_1R1W BINDTYPE storage TYPE ram IMPL auto LATENCY 2 ALLOW_PRAGMA 1}
    }
  }
  {SRCNAME conv3_Pipeline_1 MODELNAME conv3_Pipeline_1 RTLNAME srcnn_conv3_Pipeline_1}
  {SRCNAME conv3_Pipeline_VITIS_LOOP_106_1_VITIS_LOOP_107_2_VITIS_LOOP_108_3 MODELNAME conv3_Pipeline_VITIS_LOOP_106_1_VITIS_LOOP_107_2_VITIS_LOOP_108_3 RTLNAME srcnn_conv3_Pipeline_VITIS_LOOP_106_1_VITIS_LOOP_107_2_VITIS_LOOP_108_3
    SUBMODULES {
      {MODELNAME srcnn_mul_4ns_6ns_9_1_1 RTLNAME srcnn_mul_4ns_6ns_9_1_1 BINDTYPE op TYPE mul IMPL auto LATENCY 0 ALLOW_PRAGMA 1}
      {MODELNAME srcnn_mul_5ns_19ns_23_1_1 RTLNAME srcnn_mul_5ns_19ns_23_1_1 BINDTYPE op TYPE mul IMPL auto LATENCY 0 ALLOW_PRAGMA 1}
      {MODELNAME srcnn_ama_addmuladd_9ns_5ns_5ns_5ns_12_4_1 RTLNAME srcnn_ama_addmuladd_9ns_5ns_5ns_5ns_12_4_1 BINDTYPE op TYPE all IMPL dsp48 LATENCY 3 ALLOW_PRAGMA 1}
    }
  }
  {SRCNAME conv3_Pipeline_3 MODELNAME conv3_Pipeline_3 RTLNAME srcnn_conv3_Pipeline_3}
  {SRCNAME conv3 MODELNAME conv3 RTLNAME srcnn_conv3
    SUBMODULES {
      {MODELNAME srcnn_mul_5ns_8ns_12_1_1 RTLNAME srcnn_mul_5ns_8ns_12_1_1 BINDTYPE op TYPE mul IMPL auto LATENCY 0 ALLOW_PRAGMA 1}
      {MODELNAME srcnn_conv3_input_fm_buffer_RAM_AUTO_1R1W RTLNAME srcnn_conv3_input_fm_buffer_RAM_AUTO_1R1W BINDTYPE storage TYPE ram IMPL auto LATENCY 2 ALLOW_PRAGMA 1}
      {MODELNAME srcnn_conv3_output_fm_buffer_0_RAM_AUTO_1R1W RTLNAME srcnn_conv3_output_fm_buffer_0_RAM_AUTO_1R1W BINDTYPE storage TYPE ram IMPL auto LATENCY 2 ALLOW_PRAGMA 1}
    }
  }
  {SRCNAME srcnn MODELNAME srcnn RTLNAME srcnn IS_TOP 1
    SUBMODULES {
      {MODELNAME srcnn_fadd_32ns_32ns_32_4_full_dsp_1 RTLNAME srcnn_fadd_32ns_32ns_32_4_full_dsp_1 BINDTYPE op TYPE fadd IMPL fulldsp LATENCY 3 ALLOW_PRAGMA 1}
      {MODELNAME srcnn_fmul_32ns_32ns_32_3_max_dsp_1 RTLNAME srcnn_fmul_32ns_32ns_32_3_max_dsp_1 BINDTYPE op TYPE fmul IMPL maxdsp LATENCY 2 ALLOW_PRAGMA 1}
      {MODELNAME srcnn_fcmp_32ns_32ns_1_2_no_dsp_1 RTLNAME srcnn_fcmp_32ns_32ns_1_2_no_dsp_1 BINDTYPE op TYPE fcmp IMPL auto LATENCY 1 ALLOW_PRAGMA 1}
      {MODELNAME srcnn_input_r_m_axi RTLNAME srcnn_input_r_m_axi BINDTYPE interface TYPE adapter IMPL m_axi}
      {MODELNAME srcnn_params_m_axi RTLNAME srcnn_params_m_axi BINDTYPE interface TYPE adapter IMPL m_axi}
      {MODELNAME srcnn_output_r_m_axi RTLNAME srcnn_output_r_m_axi BINDTYPE interface TYPE adapter IMPL m_axi}
      {MODELNAME srcnn_control_s_axi RTLNAME srcnn_control_s_axi BINDTYPE interface TYPE interface_s_axilite}
    }
  }
}
