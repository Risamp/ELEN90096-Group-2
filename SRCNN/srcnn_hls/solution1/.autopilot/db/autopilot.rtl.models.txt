srcnn_mux_3_2_12_1_1
srcnn_mux_3_2_24_1_1
srcnn_mul_4ns_6ns_9_1_1
srcnn_mul_3ns_8ns_10_1_1
srcnn_urem_8ns_8ns_8_12_1
srcnn_urem_9ns_8ns_9_13_1
srcnn_urem_4ns_3ns_2_8_1
srcnn_mux_2_1_32_1_1
srcnn_mul_4ns_8ns_11_1_1
srcnn_mul_8ns_10ns_17_1_1
srcnn_mul_9ns_11ns_19_1_1
srcnn_mul_5ns_8ns_11_1_1
srcnn_mul_24s_12s_35_1_1
srcnn_mac_muladd_24s_12s_35ns_35_4_1
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_urem_9ns_8ns_9_13_seq_1
srcnn_mul_6ns_19ns_24_1_1
srcnn_conv1_p_ZZ5conv1PA255_A255_8ap_fixedILi24ELi3EL9ap_q_mode5EL9ap_o_mode3ELi0EEbkb
srcnn_conv1_p_ZZ5conv1PA255_A255_8ap_fixedILi24ELi3EL9ap_q_mode5EL9ap_o_mode3ELi0EEeOg
srcnn_conv1_conv1_ap_fixed_255_255_ap_fixed_1_9_9_ap_fixed_ap_fixed_255_2_1_RAM_AUThbi
srcnn_mul_7ns_19ns_25_1_1
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_mul_32s_18s_49_1_1
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_mul_5ns_19ns_23_1_1
srcnn_conv2_input_fm_buffer_RAM_2P_LUTRAM_1R1W
srcnn_conv2_weight_buffer_RAM_AUTO_1R1W
srcnn_conv2_output_fm_buffer_RAM_AUTO_1R1W
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_mul_6ns_6ns_11_1_1
srcnn_urem_9ns_9ns_9_13_1
srcnn_mux_2_1_24_1_1
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_flow_control_loop_pipe_sequential_init
srcnn_conv3_conv3_ap_fixed_255_255_ap_fixed_32_5_5_ap_fixed_ap_fixed_255_3_RAM_AUTOjbC
srcnn_conv3_weight_buffer_0_RAM_AUTO_1R1W
srcnn_conv3_conv3_ap_fixed_255_255_ap_fixed_32_5_5_ap_fixed_ap_fixed_255_1_RAM_AUTOlbW
srcnn_i1_m_axi
srcnn_w1_m_axi
srcnn_i2_m_axi
srcnn_w2_m_axi
srcnn_i3_m_axi
srcnn_w3_m_axi
srcnn_o_m_axi
srcnn_control_s_axi
conv1_Pipeline_OUT_ROW_COL
conv1_Pipeline_RELU
conv1_Pipeline_3
conv1_Pipeline_CLEAR_BH_BW
conv1
conv2_Pipeline_LOAD_INPUT_BH_L
conv2_Pipeline_LOAD_WEIGHTS_L
conv2_Pipeline_OUT_ROW_COL
conv2_Pipeline_RELU
conv2_Pipeline_5
conv2_Pipeline_RELU4
conv2_Pipeline_7
conv2_Pipeline_BW
conv2_Pipeline_BW5
conv2_Pipeline_BW6
conv2
load_input_buffer_c3
conv3_Pipeline_WEIGHTI_WEIGHTK_L
conv3_Pipeline_IN_ROW_COL
conv3_Pipeline_RELU
conv3_Pipeline_4
conv3_Pipeline_RELU1
conv3_Pipeline_6
conv3_Pipeline_CLEARW
conv3_Pipeline_CLEARW2
conv3_Pipeline_CLEARW3
conv3
srcnn
