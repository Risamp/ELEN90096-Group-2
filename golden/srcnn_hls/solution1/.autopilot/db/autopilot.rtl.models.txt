srcnn_fadd_32ns_32ns_32_4_full_dsp_1
srcnn_fmul_32ns_32ns_32_3_max_dsp_1
srcnn_fcmp_32ns_32ns_1_2_no_dsp_1
srcnn_conv1_output_ftmap_RAM_AUTO_1R1W
srcnn_conv2_output_ftmap_RAM_AUTO_1R1W
srcnn
